src/components/GradeItemAccordion.tsx

```diff
@@ -8,51 +8,98 @@ export interface GradeItemAccordionInterface {
 	updateItem: (item: CourseItem) => void;
 }
 
-const Accordion = styled.tr`
-	background-color: #ffffff;
+const Accordion = styled.div`
 	color: #4f4f4f;
-	padding: 2em;
-	justify-content: center;
-	line-height: 8em;
-	transition: line-height 14em ease-out;
-	border: #ffffff;
-	text-decoration: none;
-	border: none;
+
+	display: grid;
+	grid-template-columns:
+		minmax(75px, 1fr) minmax(200px, 5fr) minmax(200px, 5fr)
+		minmax(200px, 5fr) minmax(200px, 5fr);
+	grid-template-rows: 100px;
+	grid-template-areas: 'arrow item weight grade duedate';
 `;
-const RowComponent = styled.td`
-	border: #ffffff;
+const RowComponent = styled.div`
 	font-weight: bold;
 	font-size: large;
 	border: none;
-	margin: 1.5em;
+
+	background-color: #ffffff;
 `;
 const DropDownArrow = styled.button`
 	border: none;
 	outline: none;
-	background-color: #ffffff;
 	font-size: 2em;
+	background-color: #ffffff;
 `;
-const AccordionExtended = styled.tr`
+const AccordionExtended = styled.div`
 	font-weight: bold;
 	font-size: large;
 	color: #4961e1;
 	border: none;
-`;
-const AccordionExtendedComponent = styled.td`
 	background-color: #ffffff;
+
+	display: grid;
+	grid-template-columns:
+		minmax(75px, 1fr) minmax(200px, 5fr) minmax(200px, 5fr)
+		minmax(200px, 5fr) minmax(200px, 5fr);
+	grid-template-rows: 75px;
+	grid-template-areas: 'arrow item weight grade duedate';
+`;
+
+const AccordionExtendedComponent = styled.div`
 	border: none;
+	padding-right: 25px;
 `;
 
 const DeleteButton = styled(DelButton)`
-	background-color: #ffffff;
 	color: #b80f0a;
-	margin-right: 2em;
+	width: auto;
+	font-weight: bold;
+	padding-left: 5px;
+	padding-right: 5px;
 `;
 
 function submit() {
 	console.log('changed');
 }
 
+function monthToString(month: number) {
+	switch (month) {
+		case 0:
+			return 'January';
+		case 1:
+			return 'February';
+		case 2:
+			return 'March';
+		case 3:
+			return 'April';
+		case 4:
+			return 'May';
+		case 5:
+			return 'June';
+		case 6:
+			return 'July';
+		case 7:
+			return 'August';
+		case 8:
+			return 'September';
+		case 9:
+			return 'October';
```

Comment:
Why can't we use the default date string?

Reply:
The date string returns something like "weekday month date year" and in figma it's there as "month date". I like the design of the latter better as I find it more clean, but if people like the former I can change it back. This specific function (or something similar) would be required because .getmonth() returns a value from 0 to 11.